ui/src/app.js

```diff
@@ -250,7 +250,11 @@ function processData (data) {
       chooseCoin(data.coins, data.twoWayMode)
       break
     case 'smsVerification':
-      smsVerification(data.threshold)
+      if (data.hasPhone) {
+        idVerification()
```

Comment:
Why are we going to idVerification in the case of smsVerification? Shouldn't we have an idVerification case?